src/BlazorComponents/ChartJS/ChartJSChart.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace BlazorComponents.ChartJS
 {
-    public class ChartJSChart<T> where T: ChartJsDataset
+    public class ChartJSChart<T> where T: ChartJsDataset, IChart
```

Comment:
I don't think this class is being used any more. I should probably just get rid of it.

Reply:
I will have a look at it.